Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/CONNECTSamlAssertionValidator.java

```diff
@@ -242,12 +255,12 @@ private static void checkAssertion(SamlAssertionWrapper assertion, String confir
         throws WSSecurityException {
         if (OpenSAMLUtil.isMethodHolderOfKey(confirmMethod)) {
             if (assertion.getSubjectKeyInfo() == null) {
-                LOG.debug("There is no Subject KeyInfo to match the holder-of-key subject conf method");
+                LOG.info("There is no Subject KeyInfo to match the holder-of-key subject conf method");
```

Comment:
Is "info" really the correct logging level here?  An exception is thrown next line -- for the exact reason stated in the log message.